.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "devcontainer-fluent-bit",
+	"image": "mcr.microsoft.com/devcontainers/base:jammy",
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	"features": {
+		"ghcr.io/devcontainers/features/docker-in-docker:2": {
+			"version": "latest",
+			"moby": true,
+			"dockerDashComposeVersion": "v1"
+		}
+	},
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	"forwardPorts": [
+		2020
```

Comment:
Why do we need the port?

Reply:
Developer are should be able to use in him/her local the application out of devcontainer. Maybe, a local program send request to http input of fluent-bit. It is useful.

Reply:
2020 is typically the web server though, not an input. Again, let's add a comment at least.

Reply:
that's right, maybe we can add comment lines and update the documentation about how to use tcp/udp inputs.